ibridges/__main__.py

```diff
@@ -152,6 +162,72 @@ def _list_coll(session: Session, remote_path: IrodsPath):
         raise ValueError(f"Irods path '{remote_path}' is not a collection.")
 
 
+def ibridges_setup():
+    """Use templates to create an iRODS environment json."""
+    parser = argparse.ArgumentParser(
+        prog="ibridges setup",
+        description="Tool to create a valid irods_environment.json"
+    )
+    parser.add_argument(
+        "server_name",
+        help="Server name to create your irods_environment.json for.",
+        type=str,
+        default=None,
+        nargs="?"
+    )
+    parser.add_argument(
+        "--list",
+        help="List all available server names.",
+        action="store_true"
+    )
+    parser.add_argument(
+        "-o", "--output",
+        help="Store the environment to a file.",
+        type=Path,
+        default=DEFAULT_IENV_PATH,
+        required=False,
+    )
+    parser.add_argument(
+        "--overwrite",
+        help="Overwrite the irods environment file.",
+        action="store_true"
+    )
+    args = parser.parse_args()
+    env_providers = get_environment_providers()
+    if args.list:
+        if len(env_providers) == 0:
+            print("No server information was found. To use this function, please install a plugin"
+                  " such as:\n\nhttps://github.com/UtrechtUniversity/ibridges-servers-uu"
```

Comment:
Very nice!!!! 
Being a rookie in that feature just employing my python skills ... it works!